icons/glyph_attention.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><defs><style>.cls-1{fill:#3c3c3b;}</style></defs><title>glyph_attention</title><g id="glyph_attention"><path class="cls-1" d="M99.05,88.91,71.27,40.8a5.84,5.84,0,0,0-10.12,0L33.37,88.91a5.85,5.85,0,0,0,5.06,8.77H94A5.85,5.85,0,0,0,99.05,88.91ZM69,88.26H63.47V82.77H69ZM69,78H63.47V56.24H69Z"/></g></svg>
```

Comment:
If we remove `<style>.cls-1{fill:#3c3c3b;}</style>` and the `class="cls-1" ` then it's simpler to set the color from outside the svg...

```css
.fill-current {
  fill: currentColor
}
```

We can shave a few bytes and remove the id where they aren't being reference as well